Assignment.md

```diff
@@ -9,34 +9,70 @@
 1. Write a `for` loop that will iterate from `0` until `10`.
 1. For each iteration of the `for` loop, it will multiply the number by `9` and log the result (e.g. `"2 * 9 = 18"`).
 
+console.log("starting loop ...");
 **Bonus:**
 
 - Use a nested for loop to show the tables for every multiplier from 1 to 10 (100 results total)
 
-```js
+````js
 // Your Answer
-```
+`console.log("starting loop ...");
```

Comment:
Nice idea on using an opening and closing print statement to see if the loop works as expected.